analyzers/tests/SonarAnalyzer.UnitTest/Rules/RedundantNullCheckTest.cs

```diff
@@ -26,50 +26,57 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class RedundantNullCheckTest
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.RedundantNullCheck>();
+        private readonly VerifierBuilder codeFixbuilderCS = new VerifierBuilder<CS.RedundantNullCheck>().WithCodeFix<CS.RedundantNullCheckCodeFix>();
+
         [TestMethod]
         public void RedundantNullCheck_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\RedundantNullCheck.cs", new CS.RedundantNullCheck());
+            builderCS.AddPaths("RedundantNullCheck.cs").Verify();
 
 #if NET
```

Comment:
Why there are two preprocessor conditional directives? Should we move all the methods inside one?